src/GlobalStyles.js

```diff
@@ -5,18 +5,15 @@ const styled = {createGlobalStyle};
 
 export default styled.createGlobalStyle`
   * {
+    margin: 0;
```

Comment:
I would like to suggest to apply `margin, padding and color` to the `body { ... /  }`
If you want to read more about the Universal Selector I can recommend [**this article**] 🙂(https://dev.to/clairecodes/my-misconceptions-about-the-universal-selector-in-css-4ngm)